src/components/input-field/input-field.tsx

```diff
@@ -462,7 +473,37 @@ export class InputField {
             );
         }
 
-        if (trailingIcon) {
+        if (
+            !this.isInvalid() &&
+            this.showLink &&
+            ['email', 'tel', 'url'].includes(this.type)
```

Comment:
Maybe move this to a separate function? How about `hasLink` or something like that?

Reply:
⚡